### PR TITLE
Update the Linux kernel Conan dependency

### PR DESCRIPTION
**Pull request by DavidZemon**
The linux version change should be reasonably self explanatory. The script is something I've been using for a long time, but thought it would be good to add to the repo. The way I use it is by cloning the repo on my host and then creating a symlink to the script in my `~/bin` directory

### CODE REVIEW

File: conan/sitara_profile [hunk 1]
```diff
@@ -1,5 +1,5 @@
 [build_requires]
-linux/4.4.36@wsbu/stable
+linux/4.9.59@wsbu/stable
```
Comment (mattcaron-redlion) [inline conan/sitara_profile:2]: How do we propose to handle that multiple targets use this docker image but not all of them use the same kernel?

**Review by mattcaron-redlion — APPROVED** [review]

File: conan/sitara_profile (hunk 1 above)
Comment (DavidZemon) [reply conan/sitara_profile:2]: At the point that we start supporting multiple platforms, we'll remove the symlink for the default profile (`~/.conan/profiles/default` -> `~/.conan/profiles/sitara`) and force users to select a specific profile for each invocation.

I'm not married to this solution, but it should work.

Comment (mattcaron-redlion) [reply conan/sitara_profile:2]: That is already the case. I mean, conan may not yet support it, but we have both s5t and nglsw and they are on different branches.

Comment (DavidZemon) [reply conan/sitara_profile:2]: Sorry, I wasn't specific enough. I meant "start supporting multiple platforms on Conan."
Right now, as far as I know, no one but me and Lance are using Conan, so there's no need for anything other than s5t.

Comment (mattcaron-redlion) [reply conan/sitara_profile:2]: Okedoke.

**Pull request merged**